xmppserver/src/main/webapp/pubsub-node-summary.jsp

```diff
@@ -23,6 +22,9 @@
 <% webManager.init(request, response, session, application, out ); %>
 
 <%  // Get parameters
+    int sortOrder = ParamUtils.getIntParameter(request, "sortOrder", webManager.getPageProperty("pubsub-node-summary", "console.sortOrder", PubsubNodeResultFilter.ASCENDING));
```

Comment:
(probably not as part of this MR, but...)

I wonder if there could/should be static methods/classes with ListPager that will fetch the current "stuff" - filters, page number,  sort order, etc. off a submitted form - otherwise everyone using it will end up fetching basically the same information off - just from different field names depending on the exact usage.

Could/should have been written with the original code, but hindsight ...

Reply:
You had me at 'probably not as part of this MR' ;)